### PR TITLE
X2-6853 : Not able to search traveler in quick search with valid booking

### PR DESCRIPTION
**Pull request by ZhyvotenkoOleksandr**

### CODE REVIEW

File: src/components/Search.jsx [hunk 1]
```diff
@@ -123,7 +124,7 @@ export const Search = ({
 
     // Show dropdown only when `isOpen` is set to `true` and there are items in the list.
     const open = (isOpen || !canClose || shouldStayOpen) && itemList.length > 0 && !shouldHideMenu;
-    const noResultFound = open && !isLoading && itemList.length <= 1;
+    const noResultFound = open && !isLoading && itemList.length <= 1 && inputValue.length < minChars;
```
Comment (KarthickXola) [inline src/components/Search.jsx:127]: Did we need to show noResultFound, even if the inout is less than minChars ?

File: src/components/Search.jsx [hunk 2]
```diff
@@ -190,7 +191,13 @@ export const Search = ({
                                               highlightedIndex === index ? "bg-blue-light text-white" : "",
                                           )}
                                       >
-                                          Show all results for <strong>{inputValue}</strong>
+                                          {inputValue.length >= minChars ? (
+                                              <>
+                                                  Show all results for <strong>{inputValue}</strong>
+                                              </>
+                                          ) : (
+                                              `Enter at least ${minChars} characters to begin search`
```
Comment (KarthickXola) [inline src/components/Search.jsx:199]: Did we need to check this if minChars is not set ie. it is 0 ?

File: src/components/Search.jsx (hunk 1 above)
Comment (ZhyvotenkoOleksandr) [reply src/components/Search.jsx:127]: Hide 'No results found' when input value length is less then minChars

**Review by KarthickXola — APPROVED** [review]

File: src/components/Search.jsx [hunk 3]
```diff
@@ -190,7 +191,13 @@ export const Search = ({
                                               highlightedIndex === index ? "bg-blue-light text-white" : "",
                                           )}
                                       >
-                                          Show all results for <strong>{inputValue}</strong>
+                                          {inputValue.length >= minChars ? (
```
Comment (rushi) [inline src/components/Search.jsx:194]: This code has gotten messy. We need to clean this up _later_

**Review by rushi — APPROVED** [review]

**Pull request merged**